mixpanel/__init__.py

```diff
@@ -343,14 +462,16 @@ class Consumer(object):
 
     :param str events_url: override the default events API endpoint
     :param str people_url: override the default people API endpoint
+    :param str groups_url: override the default groups API endpoint
     :param str import_url: override the default import API endpoint
     :param int request_timeout: connection timeout in seconds
     """
 
-    def __init__(self, events_url=None, people_url=None, import_url=None, request_timeout=None):
+    def __init__(self, events_url=None, people_url=None, groups_url=None, import_url=None, request_timeout=None):
```

Comment:
It would be safest to those already depending on `Consumer` to add this new parameter to the end of the params list. Otherwise, for those using positional arguments, the value they *were* passing for `import_url` might silently get used as the `groups_url` without any indication anything had changed.

It would be nice if all the URLs were adjacent in the list, but the potential wrong-URL bug seems worse.

Reply:
Great catch, thank you!